doc/procedures/pre_commit.rst

```diff
@@ -78,6 +78,8 @@ In case of a project that already has a ``.pre-commit-config.yaml`` configuratio
 
 * Inspect the ``.pre-commit-config.yaml`` file to see if the project uses mypy or not.
 * Remove the ``.pre-commit-config.yaml`` file with ``git rm --cached .pre-commit-config.yaml``.
+* Update ``pyproject.toml`` and remove all references to pre-commit hook configuration if there are any.
```

Comment:
I worry this is too vague and also somewhat misleading. My understanding is that I am supposed to remove pytest plugin configuration, not pre-commit configuration. For example I am supposed to remove this from [tool.pytest.ini_options]:
```
addopts = "--black --flake8 --mypy --ignore-glob=*/version.py"
flake8-ignore = ["E133", "E203", "E226", "E228", "N802", "N803", "N806", "N812", "N813", "N815", "N816", "W503"]
flake8-max-line-length = 110
flake8-max-doc-length = 79
```

Reply:
Yes, those are supposed to be removed. I added that now as well. And I moved the anchor to the python page, which contains a sample `pyproject.toml` file, into this bullet.

Reply:
Perhaps say it as something like this: "remove all configuration for the linters that pre-commit runs", plus some text to describe removing addopts (since that is not quite so obviously linter configuration). That way the user knows to look for "isort" and "flake8" and such and just toss them all.

Reply:
I choose a slightly different and more verbose wording.